libdnf5/conf/vars.cpp

```diff
@@ -35,7 +35,6 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include <cstring>
 #include <filesystem>
 #include <memory>
-#include <optional>
```

Comment:
Why is `optional` getting removed here? It seems independent from the rest of the changes? Can we split this to a separate commit at least?

Reply:
Whoops, thanks. I've put it back in.